cli/internal/helm/charts/edgeless/constellation-services/charts/ccm/templates/gcp-clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if eq .Values.csp "GCP" -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:cloud-provider
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: cloud-provider
```

Comment:
Can you provide an e2e test for this?
From what I can tell, there is no ServiceAccount called `cloud-provider` in our set up

Reply:
E2E, gcp, 1.27, lb, 1:1 https://github.com/edgelesssys/constellation/actions/runs/6571600139
Or do you mean that I should add an E2E test which explicitly tests this feature?

Reply:
No, just running the e2e test is fine.
But I'm still wondering who is supposed to consume this role binding as the referenced ServiceAccount does not exist (at least in our Helm charts)

Reply:
I think it's a default service account which simply exists in K8s and some (default) permissions have been removed from it. If it does not exists anymore, we'll have to create it, but currently it seems to work.